torch/ao/quantization/fx/lstm_utils.py

```diff
@@ -72,6 +72,12 @@ def make_qconfig(obs_ctr: _PartialWrapper) -> QConfig:
         float_lstm.batch_first, float_lstm.dropout, float_lstm.bidirectional)
     quantizable_lstm.qconfig = float_lstm.qconfig
 
+    for idx in range(float_lstm.num_layers):
+        quantizable_lstm.layers[idx] = torch.ao.nn.quantizable.modules.rnn._LSTMLayer.from_float(float_lstm,
+                                                                                                 idx,
```

Comment:
Looks like there's some weird spacing here. Can you fix it before landing?

Reply:
I ran FLAKE8 linter. Didn't seem to point to any weird space. Can you point where you see?